packages/react-dom-bindings/src/server/ReactFizzConfigDOM.js

```diff
@@ -174,6 +178,30 @@ export type RenderState = {
   ...
 };
 
+type Exists = null;
+type Preloaded = [];
+// Credentials here are things that affect whether a browser will make a request
+// as well as things that affect which connection the browser will use for that request.
+// We want these to be aligned across preloads and resources because otherwise the preload
+// will be wasted.
+// We investigated whether referrerPolicy should be included here but from experimentation
+// it seems that browsers do not treat this as part of the http cache key and does not affect
+// which connection is used.
+type PreloadedWithCredentials = [
+  /* crossOrigin */ ?string,
+  /* integrity */ ?string,
+];
+
+const EXISTS: Exists = null;
+// This constant is to mark preloads that have no unique credentials
+// to convey. It should never be checked by identity and we should not
+// assume Preload values in ResumableState equal this value because they
+// will have come from some parsed input.
+const PRELOAD_SIGIL: Preloaded = [];
```

Comment:
I wouldn't necessarily call this a sigil because unlike other sigils, the referential identity of this doesn't matter. Since when it goes to/from JSON it'll be multiple copies. It's a reusable value but it doesn't imply that it has to be this specific instance to indicate that's a preload.